cmd/software/deploy.go

```diff
@@ -104,8 +106,20 @@ func deployAirflow(cmd *cobra.Command, args []string) error {
 	if isDagOnlyDeploy {
 		return DagsOnlyDeploy(houstonClient, appConfig, ws, deploymentID, config.WorkingPath, nil, true)
 	}
+
+	// Fetch the description flag value from the command flags
+	desc, err := cmd.Flags().GetString("description")
+	if err != nil {
+		return err
+	}
+
+	// If the description is not set, use GetDefaultDeployDescription to get the default
+	if desc == "" {
+		desc = utils.GetDefaultDeployDescription(cmd, args)
+	}
```

Comment:
```suggestion
	// If the description is not set, use GetDefaultDeployDescription to get the default
	if description == "" {
		description = utils.GetDefaultDeployDescription(isDagOnlyDeploy)
	}
```
nit: The description variable already holds the flag value, so we don't need to re-read and assign it to a new variable. Also along similar lines, we could simply pass `isDagOnlyDeploy` to the `GetDefaultDeployDescription` function, instead of re-reading the same value again.

Reply:
Got it!